configurations/test/env.yml.tmp

```diff
@@ -24,3 +24,10 @@ MONGO_DB_NAME: catalogue
 #MONGO_PASSWORD: password
 #MONGO_PROTOCOL: mongodb+srv
 #MONGO_USER: user
+
+# Spaces added for testing.
+# Spaces are permitted but not part of capitalization exceptions or substitutions.
+DEFAULT_CAPITALIZATION_EXCEPTIONS: ACE,  BART  ,  SMART,   EB,WB,SB,NB
+# Substitutions are in format "old => new", or "old =>+ new" if space normalization is needed.
+# Substitutions are executed in order they appear.
+DEFAULT_SUBSTITUTIONS: "@ =>+ at, [\\+&] =>+ and,   \\s*\\(.+\\)\\s* =>   , \\s*\\[.+\\]\\s* =>"
```

Comment:
Can this be made into a yml list? That way it may be more readable and there could be comments about each substitution.

Reply:
@landonreed I'd like your thoughts on this. To simplify things during implementation, I have the UI and configuration use the same format for serialization/deserialization, so if we decide to use a YAML list, the UI would become more detailed to show a list of fields, one field per substitution/exceptioon, instead of a single text field for all changes right now.

Reply:
I definitely agree on the yaml list.